### PR TITLE
Karpov/turret cable piston

### PR DESCRIPTION
**Pull request by OliverW10**

### CODE REVIEW

File: robot.py [hunk 1]
```diff
@@ -189,7 +191,7 @@ def testPeriodic(self) -> None:
                 self.indexer_control.wants_to_intake = True
                 self.intake.deployed = True
 
-        self.indexer_control.execute()
+        # self.indexer_control.execute()
```
Comment (OliverW10) [inline robot.py:194]:
```suggestion
        self.indexer_control.execute()
```

File: robot.py [hunk 2]
```diff
@@ -100,7 +103,7 @@ def teleopInit(self) -> None:
         self.auto_shoot = False
 
     def testInit(self) -> None:
-        pass
+        self.turret.target = self.turret.get_angle()
```
Comment (auscompgeek) [inline robot.py:106]: Shouldn't this be in the turret `on_enable`?

File: components/turret.py [hunk 3]
```diff
@@ -14,6 +14,8 @@ class Turret:
     motor: ctre.TalonSRX
     absolute_encoder: DutyCycleEncoder
```
Comment (auscompgeek) [inline components/turret.py:16]: nit:
```suggestion
```

File: components/turret.py [hunk 4]
```diff
@@ -105,6 +113,14 @@ def execute(self) -> None:
             self.target * self.COUNTS_PER_TURRET_RADIAN,
         )
 
+        self.cable_piston.set(not self.is_piston_extended)
```
Comment (auscompgeek) [inline components/turret.py:116]: Should this be after the if statement?

**Review by auscompgeek — APPROVED** [review]
lgtm, minor comments

**Pull request merged**